interface/forms/newpatient/common.php

```diff
@@ -86,16 +86,35 @@ function refreshIssue(issue, title) {
   s.options[s.options.length] = new Option(title, issue, true, true);
  }
 
+ <?php
```

Comment:
This is fine. Then would make it clear this is an exception (which can then be replicated in other placed where an exception is used).
1. Would place a comment above  collectValidationPageRules() call that are bypassing validateUsingPageRules
2. For the list_options entry, would make option_id set to custom_encounter#new_encounter
3. For the title entry, would make title set to custom_encounter#new_encounter (and use this in the collectValidationPageRules call below)

This way it is obvious it is an exception. And guessing that other pages in the future will need to go this route, so then will be the "standard" for those instances.